lib/shelley/test/data/cardano-node-shelley/genesis.yaml

```diff
@@ -1200,519 +1198,3 @@ initialFunds:
   6114de519ddd182962ed9ed061afb654911401415e17d0f934e3c83da3c6ee2cde: 100000000000
   613139b22b1d328000063beb4b8d59935674458f74168459d07fd4ec766328163b: 100000000000
   614f27f4008ffa5454efeb925b98f743e144f3780969d36423e7940a8ba3f5b6d3: 100000000000
-
```

Comment:
Why are these removed? Oversight?

Reply:
Thanks — oversight.

I regenerated the faucet addresses since I believe we now have the new addresses with shorter hashes. But we need to change the address generation first, for the regeneration to be effective… and indeed, make sure these are preserved.

Reply:
> But we need to change the address generation first

Indeed, we need to switch to 28-bit digest for Shelley addresses.

> make sure these are preserved.

I don't understand?

Reply:
I switched to 28 bit, and regenerated everything (including these special mnemonics) in 
eae65e2